core/controllers/admin.py

```diff
@@ -631,3 +632,21 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        if feconf.CAN_SEND_EMAILS:
+            response = {
+                'msg': 'Success! Mail sent to admin.',
+            }
+            email_manager.send_dummy_mail_to_admin()
+            self.render_json(response)
+        else:
+            response = {
```

Comment:
This case shouldn't result in a 200, it should result in a 400 error (raise InvalidInputException) and the frontend should handle that correctly. "200" is for "everything is OK".

Reply:
Done.